ui/admin/config/environment.js

```diff
@@ -13,6 +13,9 @@ const featureEditions = {
     'byow-pki-hcp-cluster-id': false,
     'byow-pki-upstream': true,
     'vault-worker-filter': false,
+    'target-multi-hop': false,
```

Comment:
Can this be changed to align with the other two flag names?

Reply:
Happy to change it. Are you thinking `target-dual-worker`? I originally thought to keep the naming slightly different since that flag is temporary versus the other two flags that are used for building the app.

Reply:
Why did we change from multi to dual?

Reply:
Giving this a little thought I think this is a good change because we could potentially have other "multi-hop" work that is not directly related to the target worker filter changes. Using `target-dual-worker` will help if that is the case and we can easily split the feature flags. I hope that makes sense.